History.md

```diff
@@ -0,0 +1,133 @@
+
+v3.0.0 / 2017-07-27
+===================
+
+  * Flush on first message (#113)
```

Comment:
(and we can copy this cleaned up version in the github releases section as well)